### ISSUE TITLE
WP 4.9 Pollyfill JS Compatablity.

### ISSUE THREAD

**Issue by milindmore22** [Automattic/amp-wp#6100]
## Bug Description

AMP Settings screen is inaccessible with WP 4.9.x versions. due to polyfill js dependencies as old version required to attach it to global windows object.

This also brings the question of if we should [drop support for older WP versions 4.9?](https://github.com/ampproject/amp-wp/issues/2431) it's been more than 2 years since Gutenberg is released.

The console error

```
Uncaught TypeError: Cannot read property 'i18n' of undefined
    at wp-api-fetch.js?ver=7645fa7ef83c5d94141bdac792ab796a:1
    at Object.2 (wp-api-fetch.js?ver=7645fa7ef83c5d94141bdac792ab796a:1)
    at r (wp-api-fetch.js?ver=7645fa7ef83c5d94141bdac792ab796a:1)
    at Object.48 (wp-api-fetch.js?ver=7645fa7ef83c5d94141bdac792ab796a:1)
    at r (wp-api-fetch.js?ver=7645fa7ef83c5d94141bdac792ab796a:1)
    at Module.<anonymous> (wp-api-fetch.js?ver=7645fa7ef83c5d94141bdac792ab796a:1)
    at Module.216 (wp-api-fetch.js?ver=7645fa7ef83c5d94141bdac792ab796a:1)
    at r (wp-api-fetch.js?ver=7645fa7ef83c5d94141bdac792ab796a:1)
    at Object.215 (wp-api-fetch.js?ver=7645fa7ef83c5d94141bdac792ab796a:1)
    at r (wp-api-fetch.js?ver=7645fa7ef83c5d94141bdac792ab796a:1)
admin.php?page=amp-options:81 

Uncaught ReferenceError: wp is not defined at admin.php?page=amp-options:81
```

## Expected Behaviour

AMP Settings screen Works as expected with the WP 4.9

## Steps to reproduce

1. Setup WP 4.9 or you can use [WP Downgrade](https://wordpress.org/plugins/wp-downgrade/) Plugin and set up 4.9
2. Install AMP if it's already not present.
3. Visit the AMP settings screen.
4. Open Browser Console and See error

## Screenshots

![image](https://user-images.githubusercontent.com/1132541/115911213-711bf880-a48b-11eb-89a0-d25754e161c5.png)

## Additional context

 - WordPress version: 4.9.x
 - Plugin version: 2.0.11
 - Gutenberg plugin version (if applicable): Nope we are going Classic 
 - AMP plugin template mode: Transional Mode
 - PHP version: 7.4
 - OS: Mac OS
 - Browser: chrome
 - Device: Macbook Air

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

**Comment by delawski** [comment]
I've done some research today and found out that the Settings page stopped working on WordPress 4.9 after releasing [AMP 2.0.5](https://github.com/ampproject/amp-wp/releases/tag/2.0.5).

I'm almost certain it has something to do with #5461. Most likely I won't have time to look into this today, but it's on my to-do list for tomorrow.

**Comment by westonruter** [comment]
@delawski This change may be the cause: https://github.com/ampproject/amp-wp/commit/05d751ace6dc2fd89fc514f69f27d19d42c1652b (via #5461).

**Comment by delawski** [comment]
@westonruter As @pierlon found out and fixed in #6119, the issue was caused by the fact that the `wp-i18n` package now depends on `wp-hooks` (see WordPress/gutenberg#27966). The PR is now ready for your review/QA and merge.

**Comment by westonruter** [comment]
**QA Passed**

On 4.9, accessing the setting screen results in:

![image](https://user-images.githubusercontent.com/134745/116320009-4d4b0200-a76c-11eb-8289-269839b8c295.png)

Upon upgrading to 2.1 on `develop`, no such error:

![image](https://user-images.githubusercontent.com/134745/116320409-0c9fb880-a76d-11eb-8cba-8c0fe52bf048.png)
